hops/hdfs.py

```diff
@@ -87,6 +88,20 @@ def project_id():
     """
     return os.environ[constants.ENV_VARIABLES.HOPSWORKS_PROJECT_ID_ENV_VAR]
 
+def project_id_as_shared(name=None):
```

Comment:
Maybe we can put this in the project.py module instead?

Reply:
Or if this is mainly for internal use we can put it in util.py

Reply:
moved to project